src/client/debug/log/LogEvent.svelte

```diff
@@ -3,17 +3,18 @@
   export let action;
   export let pinned;
   export let payload;
-  export let payloadComponent;
+  export let metadataComponent;
 
-  import CustomPayload from './CustomPayload.svelte';
+  import LogMetadata from './LogMetadata.svelte';
   import { createEventDispatcher } from 'svelte';
 
   const dispatch = createEventDispatcher();
 
   const args = action.payload.args;
   const renderedArgs = typeof args === 'string' ? args : (args || []).join(',');
   const playerID = action.payload.playerID;
-  let actionType; 
+  const metadata = action.payload.metadata
+  let actionType;
```

Comment:
I'm not sure if I found the file where you found that interface. I changed the interface in the main types file to be
``` typescript
export interface LogEntry {
  action:
    | ActionShape.MakeMove
    | ActionShape.GameEvent
    | ActionShape.Undo
    | ActionShape.Redo;
  _stateID: number;
  turn: number;
  phase: string;
  redact?: boolean;
  automatic?: boolean;
  metadata?: any; // <= Adding this
}
```

And also changed the `Log.svelte` to pass `metadata` directly instead of `payload`

Reply:
Perfect! Sorry, the interface I typed out there was just a way of showing the duplicate `payload` keys. You found the actual interface I was referring to correctly.